src/pages/Members/Members.scss

```diff
@@ -0,0 +1,47 @@
+@use '../../styles/variables' as var;
+@use '../../styles/mixins' as mix;
+
+.members-section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4rem 6rem 8rem 6rem;
+
+  .members-section-left {
+    max-width: 50%;
+    padding-left: 4rem;
+  }
+  .members-section-right {
+    width: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .members-illustration {
+    width: 40%;
+  }
+
+  .members-description {
+    max-width: 100%;
+    font-weight: var.$semibold;
+    padding-top: 3rem;
+  }
+}
+
+.heading2-underlined {
+  @include mix.heading2;
+  position: relative;
+  display: inline-block;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0.2em; // control overlap with text
+    left: -2.5%;
+    width: 105%;
+    height: 0.4em; // thickness of the underline
+    background-color: var.$secondary-color;
+    z-index: -1;
+  }
```

Comment:
This is the second time I see. This should also be part of the Heading component. (even you wouldn't have the Heading, it should be somehow abstracted. You cannot copy the same 8 lines of CSS everywhere you would have the underline)